### DISCUSSION TITLE
New "Outdoors Style" - comment on rendering

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#7123] [Q&A]
Great to see a new Style. 

But could it be a bit more visible...
![Screenshot 2024-01-09 at 12 55 16](https://github.com/organicmaps/organicmaps/assets/58628197/02b6aaf3-fc07-441d-b1fd-8ff369d3082e)
Perhaps we could have a _number_ of such styles with a **bold** and _light_ rendering of lines and points. The user could then select olight1 or olight2 or olight3 depending on their preference.

Thanks again to the wonderful coders that make OM so great !!

**Comment by pastk** [comment]
Thanks for your feedback!

Fences visibility was improved a bit (now they're appear earlier / at more overview zooms) in https://github.com/organicmaps/organicmaps/pull/6970

Regarding making fences as contrast as on your screenshot... I'd argue that hiking trails should be more prominent/visible compared to fences (or other long obstacles like cliffs) as usually people follow the trails. And usually its assumed that if there is a trail displayed then it should be accessible/walkable.
So its more important to distinguish and highlight unexpected accessibility things like locked gates, stiles, private trail sections - this needs more work indeed.

**Reply by pastk** [reply]
Would be great to create a separate issue for gates/stiles rendering and analyse which types of them should be rendered how..

**Comment by pwb999** [comment]
Great to see the progress of the Outdoors style.
In the UK very few paths are marked in any way. 
The walker has to make their own decision as to its suitability dependent on experience, fitness and weather. OSM based maps are very useful as a guide but are not viewed as definitive. Very few have any SAC Tags and are not verified as to their suitability by any official body.
It is widely recommended to follow any linear physical features in the event of bad weather and fog. 
For that reason walls/fences/streams and paths can be followed from high to safer lower ground. Gates and Stiles are NOT marked on the official Ordnance Survey maps. 
![Screenshot 2024-01-17 at 14 55 56](https://github.com/organicmaps/organicmaps/assets/58628197/0bfb2969-1156-419b-b647-043198d6e7f0)

All of these features aid a walkers safe navigation.
Many would argue that as walls tend to more permanent features it is better to use that rather than an indistinct path that may or may not be the correct one.
I therefore download such information from OSM via overpass to create the overlays of what I consider to be useful information. 
Since walls and fences are so important I render these as width = 2 in the KML file. 
Stiles are marked as Red pins; Gates as Green.
I print my intended route off as a A4 sheet so that I can navigate both using a paper map and iPhone. Having bolder lines and points help me to focus on the important details.
